apps/meteor/app/livechat/server/api/v1/contact.ts

```diff
@@ -166,6 +167,25 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'omnichannel/contacts.checkExistence',
+	{ authRequired: true, permissionsRequired: ['view-livechat-contact'], validateParams: isGETOmnichannelContactsCheckExistenceProps },
+	{
+		async get() {
+			const { contactId, visitor, email, phone } = this.queryParams;
+			const filter = {
```

Comment:
Just making sure we have a proper index that covers at least the base usecase of the endpoint 👀 
do we?